auth/libraries/pip-auth/authlib/hashing_handler.py

```diff
@@ -1,20 +1,46 @@
-from typing import Optional
+import base64
+import hashlib
 
 import argon2
 
 
+SHA512_HASHED_LENGTH = 152
+ARGON2_HASH_START = "$argon2"
+
+
 class HashingHandler:
-    def hash(self, to_hash: str, hash_reference: Optional[str] = None) -> str:
+    def hash(self, to_hash: str) -> str:
         ph = argon2.PasswordHasher()
         return ph.hash(to_hash)
 
-    def verify(self, hash: str, password: str) -> bool:
-        ph = argon2.PasswordHasher()
-        try:
-            return ph.verify(hash, password)
-        except (
-            argon2.exceptions.VerifyMismatchError,
-            argon2.exceptions.VerificationError,
-            argon2.exceptions.InvalidHashError,
-        ):
+    def is_equals(self, toHash: str, toCompare: str) -> bool:
```

Comment:
Can we use `equals` oder `is_equal` here?

Reply:
Theoretically we could, but the auth service uses this naming everywhere else as well

Reply:
Renamed all python methods